test/e2e/network_create_test.go

```diff
@@ -475,13 +480,13 @@ var _ = Describe("Podman network create", func() {
 		ncFail := podmanTest.Podman([]string{"network", "create", "--subnet", "fd00:4:4:4:4::/64", "--ipv6", netName2})
 		ncFail.WaitWithDefaultTimeout()
 		defer podmanTest.removeNetwork(netName2)
-		Expect(ncFail).To(ExitWithError())
+		Expect(ncFail).To(ExitWithError(125, "subnet fd00:4:4:4::/64 is already used on the host or by another config"))
 	})
 
 	It("podman network create with invalid network name", func() {
-		nc := podmanTest.Podman([]string{"network", "create", "foo "})
```

Comment:
Extra space was making impossible to strcmp() error message. I could not figure out why.